### PR TITLE
New post: Kotlin for Python developers

### PR DESCRIPTION
**Pull request by aasmundeldhuset**

### CODE REVIEW

File: src/styles/post-template.less [hunk 1]
```diff
@@ -251,6 +252,10 @@ body {
             border-color: @wtfColor;
         }
 
+        &.team-content-platform {
```
Comment (csilvers) [inline src/styles/post-template.less:255]: Looks like you need to copy this definition at around line 480 as well.

**Review by csilvers — APPROVED** [review]
Looks great!  Excited to see this go out once the styling changes are complete.

File: src/styles/post-template.less (hunk 1 above)
Comment (dangoor) [reply src/styles/post-template.less:255]: Good catch! It looks like `wtf` and `mobile` are also missing entries in that part of the file. @aasmundeldhuset Either you can copy those two while you're at it, or I can do it in a separate commit after this one is done.

**Review by dangoor — APPROVED** [review]

File: src/styles/post-template.less (hunk 1 above)
Comment (aasmundeldhuset) [reply src/styles/post-template.less:255]: Thanks - fixed!

**Pull request merged**